docs/docs/quickstart/index.md

```diff
@@ -0,0 +1,202 @@
+---
+sidebar_position: 2
+---
+import Figure from '../figure'
+
+# Quick-Start
+
+This document only explains the bare minimum needed to know to get from an API description to a functional *Unreal Engine* plugin.
```

Comment:
One suggestion: "This document only explains the bare minimum needed to know to get from an API description to a functional *Unreal Engine* plugin." => "The Quick-Start guide explains how to, in few steps, get from API to functional Unreal Engine plugin"